README.md

```diff
@@ -23,21 +23,101 @@
 
 ## Introduction
 
-Jujutsu is a
-[Git-compatible](https://martinvonz.github.io/jj/latest/git-compatibility)
-[DVCS](https://en.wikipedia.org/wiki/Distributed_version_control). It combines
-features from Git (data model,
-[speed](https://github.com/martinvonz/jj/discussions/49)), Mercurial (anonymous
-branching, simple CLI [free from "the
-index"](https://martinvonz.github.io/jj/latest/git-comparison#the-index),
-[revsets](https://martinvonz.github.io/jj/latest/revsets), powerful
-history-rewriting), and Pijul/Darcs ([first-class
-conflicts](https://martinvonz.github.io/jj/latest/conflicts)), with features not
-found in most of them
-([working-copy-as-a-commit](https://martinvonz.github.io/jj/latest/working-copy),
-[undo functionality](https://martinvonz.github.io/jj/latest/operation-log),
-automatic rebase, [safe replication via `rsync`, Dropbox, or distributed file
-system](https://martinvonz.github.io/jj/latest/technical/concurrency)).
+Jujutsu is a powerful [version control system](https://en.wikipedia.org/wiki/Version_control)
+for software projects. You use it to get a copy of your code, track changes
+to the code, and finally publish those changes for others to see and use.
+It is designed from the ground up to be easy to use—whether you're new or
+experienced, working on brand new projects alone, or large scale software
+projects with large histories and teams.
+
+Jujutsu is unlike most other systems, because internally it abstracts the user
+interface and version control algorithms from the *storage system* used to
+serve your content. This allows it to serve as a VCS with many possible physical
+storage backends, that may have their own data or networking models—like
+[Mercurial] or [Breezy], or hybrid systems like Google's cloud-based system,
+"Code in the Cloud".
```

Comment:
CitC actually stands for "Clients in the Cloud", but CitC is not very relevant to the storage systems we talk about here. CitC is just a storage system for the working copy (we'll implement https://github.com/martinvonz/jj/blob/main/lib/src/working_copy.rs for CitC at Google). I think it's better to just drop "Code in the Cloud" here.

Reply:
The sentence sort of trails off at that point? But I wanted to mention it since it's a much more unique design point than just "random other version control system backend." I just reworked the paragraph a little to be less pushy on the 'storage' bit, and called it "Piper/CitC" instead, which I think has a bunch of pre-existing references you could Google(!) or look up. Let me know if you think that's fine.

Reply:
I think of it differently [^1], but I think what you have is probably a good description for new users.

[^1]: I think of what we call "commit cloud" (a term copied from Meta) - i.e. our Spanner database and our server  - as the storage backend. The integration with Piper is honestly less important, in the sense that the commit database could live without Piper (but the integration with Piper is completely necessary in practice because that's what all other tooling integrates with).